index.html

```diff
@@ -0,0 +1,106 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body>
+
+
+    <div class="container">
+      <header class="header">
+        <h1 id="title" class="text-center"> Survey Form </h1>
+        <p  class="decrip">
+         Please fill this survey form
+        </p>
+      </header>
```

Comment:
Please kindly remove the header from the div tag.  The header always stands alone. 

Instead of using a div class for your container you can use section

Reply:
@tarikbouari  please make sure to explain why you require changes maybe it's obvious for you but not for everyone 
Explain why the section tag is better than div tag